### PR TITLE
[wallet]Select blockchain url based on testnet

### PR DESCRIPTION
**Pull request by ashishb**
### Description

Earlier, the block chain API url was hardcoded in `.env`.
Now, select it based on the testnet the app is using.

### Tested

`packages/mobile $ yarn lint && jest`

### CODE REVIEW

File: packages/mobile/src/geth/network-config.ts [hunk 1]
```diff
@@ -5,21 +5,26 @@ export default {
...
   },
   [Testnets.alfajoresstaging]: {
     nodeDir: `.${Testnets.alfajoresstaging}`,
     syncMode: SYNC_MODE_ULTRALIGHT,
+    blockchainApiUrl: 'https://alfajoresstaging-dot-celo-testnet.appspot.com/',
   },
   [Testnets.alfajores]: {
     nodeDir: `.${Testnets.alfajores}`,
     syncMode: SYNC_MODE_ULTRALIGHT,
+    blockchainApiUrl: 'https://pilot-dot-celo-testnet-production.appspot.com/',
```
Comment (annakaz) [inline packages/mobile/src/geth/network-config.ts]: alfajores?

Comment (ashishb) [reply packages/mobile/src/geth/network-config.ts]: Good catch. Should have been `https://alfajores-dot-celo-testnet-production.appspot.com/`
Fixed.

**Review by annakaz — APPROVED** [review]
LGTM!

**Review by jmrossy — APPROVED** [review]
Thanks for fixing this. 
They got mismatched in the period where we were using alfajores for all development but glad we're working our way back to a proper set of envs

**Pull request merged**